quarkus/maven-plugin/src/main/java/org/keycloak/operator/codegen/Property.java

```diff
@@ -0,0 +1,40 @@
+package org.keycloak.operator.codegen;
+
+import java.util.Objects;
+
+public class Property {
```

Comment:
I'm wondering if this type should be added to the configuration API itself. We are using a similar structure when generating docs. See https://github.com/keycloak/keycloak/blob/52d205ca916688b7fc6c77d6a319f584b6b3fd6f/docs/maven-plugin/src/main/java/org/keycloak/guides/maven/Options.java#L92.

Reply:
I was wondering if I should try to re-use the docs generation, but I have no problems in including `Property/Options` as an extra class in the `org.keycloak.quarkus.runtime.configuration` package.

Reply:
I think that would be awesome but I don't think it is a blocker.